addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -404,7 +404,7 @@ Object.assign(ComposerTextInput, {
         hasMentionSuggestionsBelowPosition: false,
     },
     props: {
-        composerViewLocalId: String,
+        localId: String,
```

Comment:
This is a bit confusing, it misleads that there is a `ComposerTextInputView` model, but it's actually `ComposerView` here.

Also it's inconsistent with some other props that haven't been renamed, e.g. `ComposerSuggestionComponent/composerViewLocalId`.

I suppose you did the rename only for the ones that use `useXToModel` that had `propNameAsReacordLocalId`.

I'm okay to do cleanup afterward, but just pointing out the inconsistency.

Reply:
I think it's the only case, I hesitated to do it but I needed to to allow removing the `propNameAsReacordLocalId`.

Other components that didn't have `propNameAsReacordLocalId`, I didn't rename to `localId` if they didn't have their own model to avoid the confusion.